packages/back-end/src/api/experiments/experiments.router.ts

```diff
@@ -3,14 +3,17 @@ import { listVisualChangesets } from "../visual-changesets/listVisualChangesets"
 import { getExperimentResults } from "./getExperimentResults";
 import { getExperiment } from "./getExperiment";
 import { listExperiments } from "./listExperiments";
+import { putExperiment } from "./putExperiment";
 
 const router = Router();
 
 // Experiment Endpoints
 // Mounted at /api/v1/experiments
 router.get("/", listExperiments);
+router.put("/", putExperiment);
 router.get("/:id", getExperiment);
 router.get("/:id/results", getExperimentResults);
+router.get("/:id", putExperiment);
```

Comment:
I think this line got added by mistake

Reply:
It will become:
- `/:id` for `PUT`
- '/` for `POST`

Reply:
Oh yeah you're right. My bad. Will fix it now.